src/native/libs/System.Globalization.Native/config-standalone.h

```diff
@@ -0,0 +1,8 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+//
+
+#pragma once
+
+#define HAVE_UDAT_STANDALONE_SHORTER_WEEKDAYS 1
```

Comment:
Should we do regular config for this instead of hardcoding?

Reply:
ok, will do regular config